scraper/code_gov/models.py

```diff
@@ -570,3 +571,58 @@ def from_doecode(klass, record):
             }
 
         return project
+
+    @classmethod
+    def from_tfs(klass, tfs_project, labor_hours=True):
+        """
+        Creates CodeGovProject object from TFS/VSTS/AzureDevOps Instance
+        """
+        project = klass()
+        project_web_url = ''
+
+        # -- REQUIRED FIELDS --
+        project['name'] = tfs_project.projectInfo.name
+
+        if 'web' in tfs_project.projectInfo._links.additional_properties:
+            if 'href' in tfs_project.projectInfo._links.additional_properties['web']:
+                # URL Encodes spaces that are in the Project Name for the Project Web URL
+                project_web_url = requote_uri(tfs_project.projectInfo._links.additional_properties['web']['href'])
+
+        project['repositoryURL'] = project_web_url
+
+        project['homepageURL'] = project_web_url
+
+        project['description'] = tfs_project.projectInfo.description
+
+        project['vcs'] = 'TFS/AzureDevOps'
+
+        project['permissions']['license'] = None
+
+        project['tags'] = []
+
+        if labor_hours:
+            logger.debug('Sorry labor hour calculation not currently supported.')
+            # project['laborHours'] = labor_hours_from_url(project['repositoryURL'])
+        else:
+            project['laborHours'] = 0
+
+        if tfs_project.projectCreateInfo.last_update_time < POLICY_START_DATE:
```

Comment:
This may be wrong elsewhere too, but should this be if `create_time < POLICY_START_DATE`  instead of `last_update_time` ?

Reply:
This is actually not a typo, but it's a weird function of the TFS python api. The field is always called last_update_time but this one is stored on the projectCreateInfo attribute (as opposed to the projectLastUpdateInfo). The name is kept as last_update_time so it stays the same as what is passed back by the API.